pkg/controller/controller_broker.go

```diff
@@ -17,12 +17,13 @@ limitations under the License.
 package controller
 
 import (
-	stderrors "errors"
 	"fmt"
 	"time"
 
 	"github.com/golang/glog"
 
+	stderrors "errors"
```

Comment:
I believe this was an accidental move of import?
Also, probably should remove a blank line above (after `glog`) to avoid this `gofmt`'s weird behavior in future.

Reply:
done, I saw and meant to fix on initial commit.  Thanks.